src/main/java/org/jabref/model/groups/GroupHierarchyType.java

```diff
@@ -29,4 +29,15 @@ public static GroupHierarchyType getByNumberOrDefault(int type) {
             return INDEPENDENT;
         }
     }
+
+    public String getFormattedName() {
+        if (this == INDEPENDENT) {
+            return "Independent";
+        } else if (this == REFINING) {
+            return "Intersection";
+        }
+        else {
+            return "Union";
+        }
+    }
```

Comment:
This should be implemented as an constructor argument for the enum e.g.  `INDEPENDENT("Independent")` and be localized. Open/closed design principle.